includes/polling/os/eatonups.inc.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$hardware = trim(snmp_get($device, 'upsIdentModel.0', '-OQv', 'UPS-MIB'), '" ');
```

Comment:
trim() is not needed, this should code should be implemented in the new style.  See LibreNMS\OS\Qnap.php